crates/wash-cli/src/up/mod.rs

```diff
@@ -272,6 +272,10 @@ pub struct WasmcloudOpts {
     /// If enabled, allows starting additional wasmCloud hosts on this machine
     #[clap(long = "multi-local")]
     pub multi_local: bool,
+
+    /// Defines the Max Execution time (in minutes) that the host runtime will execute for
+    #[clap(long = "max-execution-time", alias = "max-time", env = WASMCLOUD_MAX_EXECUTION_TIME)]
+    pub max_execution_time: Option<u64>,
```

Comment:
I'm not sure that minutes are granular enough. Should we make this seconds?
Or, even better, how about we allow a string here like `2s` for 2 seconds or `5m` for 5 minutes?

Reply:
Okay, cmd input type needs more granularity -- about allowing strings as `2s` and `5m`, 
this function would allow us to do exactly that -- any thoughts or suggestions ?
```rust
fn parse_duration(s: &str) -> Result<Duration, String> {
    if let Some(secs) = s.strip_suffix('s') {
        secs.parse::<u64>()
            .map(Duration::from_secs)
            .map_err(|e| format!("Invalid duration: {}", e))
    } else if let Some(mins) = s.strip_suffix('m') {
        mins.parse::<u64>()
            .map(|m| Duration::from_secs(m * 60))
            .map_err(|e| format!("Invalid duration: {}", e))
    } else {
        Err("Invalid duration format, use 's' for seconds or 'm' for minutes".to_string())
    }
}
```

Reply:
Based on our own baselines, we currently measure other times in terms of milliseconds which gives a pretty good level of granularity. What are your thoughts on just continuing that pattern?

So, by default, the execution timeout would be `10 * 60 * 1000` aka `600000` milliseconds. Then the environment variable could be `WASMCLOUD_MAX_EXECUTION_TIME_MS`